pkg/apis/alicloud/types_infrastructure.go

```diff
@@ -110,7 +110,9 @@ type NatGatewayConfig struct {
 type InfrastructureStatus struct {
 	metav1.TypeMeta
 
-	VPC         VPCStatus
+	VPC VPCStatus
+
+	// KeyPairName should not be used as ssh-key-pair will not generated by terraform.
```

Comment:
FYI https://github.com/golang/go/wiki/Deprecated

```suggestion
	// KeyPairName is the key pair name.
	//
	// Deprecated: the field should not be used anymore as the SSH key pair is no longer generated by terraform.
```

Reply:
I don't think this comment is resolved.